src/app/backing/BackingPage.tsx

```diff
@@ -8,11 +8,16 @@ import {
   MetricsContainer,
   PageTitleContainer,
 } from '@/components/containers'
+import { BottomActionBar } from '../../components/BottomActionBar/BottomActionBar'
+import { Button } from '@/components/Button'
+import { useState } from 'react'
 
 const NAME = 'Backing'
 export const BackingPage = () => {
   const { address } = useAccount()
 
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
Do you think we could make a context instead so that pages don't need to know about the state of the action bar?
we could then write something like:

```
const { isOpen, setIsOpen, ActionBarDrawer } = useActionBar()

...

<ActionBarDrawer>
    <BackingActionBar>
</ActionBarDrawer>
```

where ever we need an action bar. This would enforce consistency across the app.

Reply:
in fact, can't we use LayoutContext for this? After all, the bottom drawer is the same as the side bar